dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPIConfigTest.java

```diff
@@ -0,0 +1,174 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.checkpoint.fat;
+
+import static io.openliberty.checkpoint.fat.FATSuite.configureEnvVariable;
+import static io.openliberty.checkpoint.fat.FATSuite.getTestMethodNameOnly;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+
+import componenttest.annotation.CheckpointTest;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import io.openliberty.checkpoint.fat.utils.OpenAPIConnection;
+import io.openliberty.checkpoint.spi.CheckpointPhase;
+import mpOpenAPIConfig.OpenAPIConfigTestResource;
+
+/**
+ * Tests for the mpOpenAPI config element
+ */
+@RunWith(FATRunner.class)
+@CheckpointTest
+public class OpenAPIConfigTest extends FATServletClient {
+
+    private static final String SERVER_NAME = "checkpointMPOpenAPIConfig";
+
+    private static String DEFAULT_DOC_PATH = "/openapi/docs";
+    private static String DEFAULT_UI_PATH = DEFAULT_DOC_PATH + "/ui";
+
+    private static String APP_NAME = "mpOpenAPIConfigApp";
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = FATSuite.defaultMPRepeat(SERVER_NAME);
```

Comment:
Can this really repeat.  I thought the settings tested here were specific to mpOpenApi-3.1